reverseCase/reverseCase.js

```diff
@@ -3,5 +3,13 @@
  * Example: 'Hello World' -> 'hELLO wORLD'
  * Assume that each string will contain only spaces and letters.
  */
-
+const reverseCase = (str) => {
+  return str.split('').map(function(x){
```

Comment:
Well done. This is a very common solution for this algorithm. I like how neat and tidy your code is! Very easy to read and reason about.